tests/integration/sheet.js

```diff
@@ -255,14 +259,14 @@ describe('Integration: sheet', () => {
           a: {color: 'red'}
         })
         expect(sheet.toString()).to.be(
-          `@media print {\n` +
+          '@media print {\n' +
           `  .${id} {\n` +
-          `    float: left;\n` +
-          `  }\n` +
-          `}\n` +
+          '    float: left;\n' +
+          '  }\n' +
+          '}\n' +
           `.${id} {\n` +
-          `  color: red;\n` +
-          `}`
+          '  color: red;\n' +
+          '}'
```

Comment:
i dont know how did i manage to comment on outdated code. github just showed me latest changes